### DISCUSSION TITLE
[Projects beta] Automatically give new tasks a status of "todo"

### DISCUSSION THREAD

**Discussion by jdevries3133** [community/community#11778] [Projects and Issues] [answered]
It's a lot of clicks to always have to go over and set the status as I'm adding items

**Comment by peterlauri** [comment, marked as answer]
You can use Workflows to set the status of your choice of new items added :)

![image](https://user-images.githubusercontent.com/3514945/154980039-e7da5bfb-0970-4688-b171-cc7b2d930de7.png)

**Reply by jdevries3133** [reply]
Ah nice, thank you
